EasyAppointmentManager/Models/Customer.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace EasyAppointmentManager.Models
+{
+    /// <summary>
+    /// Represents a single Customer/Patient
+    /// </summary>
+    public class Customer
+    {
+        /// <summary> 
+        /// The unique identifier for the customer
+        /// </summary>
+        [Key]
+        public int CustomerId { get; set; }
+
+        /// <summary>
+		/// The Customer's legal first name
```

Comment:
Some spacing issues right here. Not required for functionality but a fix will be nice.

Reply:
I saw that and checked on my end. Nothing's abnormal. It's weird.